### PR TITLE
Make invalid data file error go to status bar, not pop exception dialog

### PR DESCRIPTION
**Pull request by danmosemsft**

### CODE REVIEW

File: src/PerfView/Dialogs/RunCommandDialog.xaml.cs [hunk 1]
```diff
@@ -544,8 +544,16 @@ private void OKButtonClick(object sender, RoutedEventArgs e)
...
-                // This also throws an exception if there are illegal chars.  
-                var fullPath = System.IO.Path.GetFullPath(App.CommandLineArgs.DataFile);
+                string fullPath;
+                try
+                {
+                    fullPath = System.IO.Path.GetFullPath(App.CommandLineArgs.DataFile);
+                }
+                catch (Exception ex)
+                {
+                    m_mainWindow.StatusBar.LogError("Invalid datafile '" + App.CommandLineArgs.DataFile + "'. " + ex.Message);
```
Comment (sharwell) [inline src/PerfView/Dialogs/RunCommandDialog.xaml.cs]: ❓ Can we still print the full exception text to the log?

Comment (danmosemsft) [reply src/PerfView/Dialogs/RunCommandDialog.xaml.cs]: It's that the path is invalid (blank, invalid characters). It's not a bug. Is it useful to write out the .NET stack for that?

Comment (sharwell) [reply src/PerfView/Dialogs/RunCommandDialog.xaml.cs]: If that's the only case where it's hit, then I'm fine with it either way.

**Comment by danmosemsft** [comment]
@sharwell can I hit merge, or does @vancem need to sign off?

**Comment by sharwell** [comment]
@danmosemsft I typically let Vance sign off on anything except test/infrastructure changes.

**Comment by vancem** [comment]
LGTM

**Pull request merged**